keras_nlp/models/deberta_v3/deberta_v3_presets.py

```diff
@@ -26,21 +36,22 @@
             "bucket_size": 256,
         },
         "preprocessor_config": {},
+        "weights_url": "https://storage.googleapis.com/keras-nlp/models/deberta_v3_extra_small_en/v1/model.h5",
+        "weights_hash": "d8e10327107e5c5e20b45548a5028619",
+        "spm_proto_url": "https://storage.googleapis.com/keras-nlp/models/deberta_v3_extra_small_en/v1/vocab.spm",
+        "spm_proto_hash": "1613fcbf3b82999c187b09c9db79b568",
+    },
+    "deberta_v3_small_en": {
```

Comment:
There seems to be a lot of code delta unrelated to the change we want to make in this PR. We should avoid making such changes. Make only those changes which are necessary for the PR